libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.scss

```diff
@@ -1,107 +1,108 @@
+        }
+    }
+    @keyframes gradiantShiftGray {
+        0% {
+            background-position: 0% 50%;
+        }
+        50% {
+            background-position: 100% 50%;
+        }
+        100% {
+            background-position: 0% 50%;
+        }
     }
-    100% {
-      background-position: 0% 50%;
-    }
-  }
-  @keyframes gradiantShiftGray {
-    0% {
-      background-position: 0% 50%;
-    }
-    50% {
-      background-position: 100% 50%;
-    }
-    100% {
-      background-position: 0% 50%;
-    }
-  }
 }
 
 .lightToGrayBackgroundGradiate {
-  background: linear-gradient(to right, #f7f7f7, #c4c4c4);
-  background-size: 400% 400%;
-  height: 15px;
-  padding: 2.5px 0 2.5px 0;
-  text-align: center;
-  color: #888888;
-  font-size: 15px;
-  font-weight: bold;
-  margin: 10px 10px 0 10px;
-
-  -webkit-animation: gradiantShift 5s ease infinite;
-  -moz-animation: gradiantShift 5s ease infinite;
-  animation: gradiantShift 5s ease infinite;
-
-  @-webkit-keyframes gradiantShift {
-    0% {
-      background-position: 0% 50%;
-    }
-    50% {
-      background-position: 100% 50%;
-    }
-    100% {
-      background-position: 0% 50%;
-    }
-  }
-  @-moz-keyframes gradiantShift {
-    0% {
-      background-position: 0% 50%;
-    }
-    50% {
-      background-position: 100% 50%;
-    }
-    100% {
-      background-position: 0% 50%;
+    background: linear-gradient(to right, #f7f7f7, #c4c4c4);
+    background-size: 400% 400%;
+    height: 15px;
+    padding: 2.5px 0 2.5px 0;
+    text-align: center;
+    color: #888888;
+    font-size: 15px;
+    font-weight: bold;
+    margin: 10px 10px 0 10px;
+    -webkit-animation: gradiantShift 5s ease infinite;
+    -moz-animation: gradiantShift 5s ease infinite;
+    animation: gradiantShift 5s ease infinite;
+    @-webkit-keyframes gradiantShift {
+        0% {
+            background-position: 0% 50%;
+        }
+        50% {
+            background-position: 100% 50%;
+        }
+        100% {
+            background-position: 0% 50%;
+        }
+    }
+    @-moz-keyframes gradiantShift {
+        0% {
+            background-position: 0% 50%;
+        }
+        50% {
+            background-position: 100% 50%;
+        }
+        100% {
+            background-position: 0% 50%;
+        }
+    }
+    @keyframes gradiantShift {
+        0% {
+            background-position: 0% 50%;
+        }
+        50% {
+            background-position: 100% 50%;
+        }
+        100% {
+            background-position: 0% 50%;
+        }
     }
-  }
-  @keyframes gradiantShift {
-    0% {
-      background-position: 0% 50%;
-    }
-    50% {
-      background-position: 100% 50%;
-    }
-    100% {
-      background-position: 0% 50%;
-    }
-  }
 }
 
 .autoLoading {
-  height: 60px;
+    height: 60px;
 }
+
 .hide-cursor {
-  color: transparent;
-  text-shadow: 0 0 0;
+    color: transparent;
+    text-shadow: 0 0 0;
 }
+
+input::-ms-clear {
+    display: none;
```

Comment:
Is this purely an autocomplete issue? I think search components had a similar clear input button